src/lando/main/models/repo.py

```diff
@@ -154,6 +167,20 @@ def path(self) -> str:
     # Use this field to enable/disable pre-landing hooks for a repo.
     hooks_enabled = models.BooleanField(default=True)
 
+    # Bring the HOOKS_CHOICES into this namespace for external reference.
+    HOOKS_CHOICES = HOOKS_CHOICES
+
+    # WARNING: Due to this binding, whenever checks are added or removed, a database
+    # migration is needed.
```

Comment:
I think if HOOKS_CHOICES is not linked to docstrings, we don't really need this comment as there's no "magic" or anything potentially unexpected that will happen here.